plugins/commitment.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: commitment
+spec:
+  version: v0.1.1
+  homepage: https://github.com/wI2L/kubectl-commitment
+  shortDescription: Compare VerticalPodAutoscaler recommendations to actual resources requests
```

Comment:
this probably won't fit `kubectl krew list` output, it'll be truncated at 50 chars.

Reply:
Got it. I wrapped the `description` field to 80 chas as indicatd in the developper guide, but missed that. Will fix it.